Resources/config/console.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+
+<container xmlns="http://symfony.com/schema/dic/services"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <services>
+        <defaults public="false" />
```

Comment:
I suggest avoiding to use the defaults here for now. The bundle currently supports version older than 3.3. Forcing us to maintain multiple versions in parallel just to use defaults for 3 private services is not worth it IMO.